providers/redis/types.go

```diff
@@ -90,8 +91,196 @@ type Config struct {
 	Limiter redis.Limiter
 }
 
+// FailoverConfig provider settings
+type FailoverConfig struct {
+	// Key prefix
+	KeyPrefix string
+
+	// Optional username.
+	Username string
+
+	// Optional password. Must match the password specified in the
+	// requirepass server configuration option.
+	Password string
+
+	// Database to be selected after connecting to the server.
+	DB int
+
+	// The sentinel master name.
+	MasterName string
+
+	// The sentinel nodes seed list (host:port).
+	Addrs []string
```

Comment:
Could you rename to `SentinelAddrs`, please?

Reply:
Sure, will do it now.